### PR TITLE
Np 9102/update verify code css

### PR DESCRIPTION
**Pull request by AlexeyGreer**

### CODE REVIEW

File: src/foam/nanos/auth/email/EmailVerificationCode.js [hunk 1]
```diff
@@ -55,18 +55,27 @@ foam.CLASS({
...
     .foam-u2-dialog-ApplicationPopup-bodyWrapper .foam-u2-detail-SectionView-verificationCodeSection {
       width: fit-content;
       align-self: center
     }
+    .foam-u2-ActionView + .foam-u2-ActionView {
+      margin-left: 0px;
+    }
+
+    .foam-u2-PropertyBorder {
+        gap: 0.6rem;
```
Comment (sarthak-marwaha) [inline src/foam/nanos/auth/email/EmailVerificationCode.js:73]: spacing

File: src/foam/nanos/auth/email/EmailVerificationCode.js [hunk 2]
```diff
@@ -55,18 +55,27 @@ foam.CLASS({
...
+    }
+
+    .foam-u2-PropertyBorder-errorText {
+        max-width: 29rem;
```
Comment (sarthak-marwaha) [inline src/foam/nanos/auth/email/EmailVerificationCode.js:77]: spacing

File: src/foam/nanos/auth/email/VerificationCodeView.js [hunk 3]
```diff
@@ -27,8 +27,28 @@
...
+
+    }
+    ^sectionView{
+      width: 25%;
+      display: flex;
+      justify-content: center;
+    }
+
+    ^title {
+        text-align:center;
```
Comment (sarthak-marwaha) [inline src/foam/nanos/auth/email/VerificationCodeView.js:42]: spacing

Comment (AlexeyGreer) [reply src/foam/nanos/auth/email/VerificationCodeView.js:42]: Yeah I got it the first time

**Comment by AlexeyGreer** [comment]
Don't merge, I broke something.

**Pull request closed without merging**